### ISSUE TITLE
How to update the editor text programmatically?

### ISSUE THREAD

**Issue by reveurguy** [steven-tey/novel#182]
I am generating some text using openAI api calls.
I want to insert that text into the editor ( ideally in markdown format, the output is in markdown format )
So how can I do that? The output is being streamed.

**Comment by arrlancore** [comment]
I have made a PR to support markdown paste, you may can reference to this PR https://github.com/steven-tey/novel/pull/189 to get idea how to handle your case.

**Comment by steven-tey** [comment]
This is blocked by #58 unfortunately 😓 

You should still be able to stream regular text in though!

Closing this as a duplicate of #58 – let's discuss this over there!
